src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_datapathavailability_alert.bicep

```
@@ -0,0 +1,339 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+targetScope = 'managementGroup'
+
+param policyLocation string = 'centralus'
+param deploymentRoleDefinitionIds array = [
+    '/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c'
+]
+
+@allowed([
+    '0'
+    '1'
+    '2'
+    '3'
+    '4'
+])
+param parAlertSeverity string = '0'
```

```suggestion
param parAlertSeverity string = '1'
```

@jfaurskov if I remember correctly, this type of alert we would classify as a 1, correct?